### ISSUE TITLE
[Masonry][Accessibility] Files: Main Navigation (Card View) - Selected state of the folder is not announced to the screen reader

### ISSUE THREAD

**Issue by majornista** [adobe/coral-spectrum#249]
### Expected Behavior

A screen reader should accurately announces the selected/unselected state of Masonry items, which in AEM will contain Card View items and folders.

### Actual Behavior

A screen reader does not accurately announce the selected/unselected state of Masonry items, which in AEM will contain Card View items and folders.

### Reproduce Scenario (including but not limited to)

https://opensource.adobe.com/coral-spectrum/examples/#masonry or any AEM UI using coral-masonry.

#### Steps to Reproduce

Described in ASSETS-6844

#### Browser name/version/os (ie Chrome Version 62.0.3202.94 (Official Build) (64-bit) MacOS)

Chrome Version 105.0.5195.125 (Official Build) (arm64) MacOS

#### Coral Spectrum version

v4.15.2
